frontend/src/app.js

```diff
@@ -27,6 +27,7 @@ const App = () => {
           <Navbar />
           <Switch>
             <Route exact path="/" component={Home} />
+            <Route exact path="/adminLOgIn" component={Admins} />
```

Comment:
Try to give meaningful names everywhere! Change the path from `adminLOgin` to `admin-setting`